build.gradle

```diff
@@ -52,6 +52,7 @@ configure(subprojects.findAll {it.name == 'auctionApp' || it.name == 'auctionBid
 		postgreSQLJdbcVersion = "9.4.1212.jre7"
 		dbcpVersion = "1.4"
 		springDataMongoVersion = "1.10.9.RELEASE"
```

Comment:
Should the mongo references in this file be removed?

Reply:
Yes.  I have updated this file.